robot/Cumulus/tests/browser/gift_entry/new_gift_functionality.robot

```diff
@@ -0,0 +1,95 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/AccountPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             API Check And Enable Gift Entry
+...             Setup Test Data
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Keywords ***
+Setup Test Data
+    [Documentation]      Creates the organiation account and household contact record
+    ...                  along with getting dates required for test.
+    &{account} =         API Create Organization Account    Name=${faker.company()}
+    ...                  BillingStreet=${faker.street_address()}
+    ...                  BillingCity=${faker.city()}
+    ...                  BillingState=California
+    ...                  BillingPostalCode=${faker.postcode()}
+    Set suite variable   &{ACCOUNT}
+    ${date} =            Get Current Date         result_format=%Y-%m-%d
```

Comment:
Please capitalize all variables consistently

Reply:
thanks for catching, updated it.